app/src/main/java/org/jellyfin/androidtv/util/InfoLayoutHelper.java

```diff
@@ -274,11 +274,11 @@ private static void addDate(@NonNull  Activity activity, BaseItemDto item, Linea
                 StringBuilder sb = new StringBuilder();
                 if (item.getPremiereDate() != null) {
                     sb.append(activity.getString(R.string.lbl_born));
-                    sb.append(new SimpleDateFormat("d MMM y").format(TimeUtils.convertToLocalDate(item.getPremiereDate())));
+                    sb.append(DateFormat.getDateFormat(activity).format(TimeUtils.convertToLocalDate(item.getPremiereDate())));
```

Comment:
Maybe we should use `getMediumDateFormat` for these? It seems like it should match closer to the previous format. https://developer.android.com/reference/android/text/format/DateFormat#getMediumDateFormat(android.content.Context)

Reply:
`getMediumDateFormat` does give better output, I've changed it. I've also changed the function to use a context instead of an activity which is preferred in android dev (although an activity is an context).